### PR TITLE
simple a2ensite/a2dissite module

### PR DESCRIPTION
**Pull request by fabfurnari**
Just a copy of the very useful apache2_module module. It manages the activation and deactivation of sites using a2ensite/a2dissite commands.
It's tested and should work fine...

### CODE REVIEW

**Comment by fabfurnari** [comment]
Don't know why Travis CI marks that code as incorrect

**Comment by robynbergeron** [comment]
Hi @fabfurnari --

Thanks for submitting this new module to Ansible Extras! This module is now in community review, a process that is open to all Ansible users. In order for this module to be approved, it must gain the following votes, which can be contributed by anyone other than the author of the new module:

“works_for_me”: If you have tested the module thoroughly, including testing of all of the module’s options, and if the module works for you, please add “works_for_me” in the comments.

“passes_guidelines”: If you have gone through the module guidelines and the module meets all of the requirements, please add “passes_guidelines” in the comments. Guidelines are available here: http://docs.ansible.com/developing_modules.html#module-checklist

“needs_revision”: If the module fails to work for you, or if it doesn’t meet guidelines, please add “needs_revision” in the comments with details about what needs to be fixed.

When a module has both “works_for_me” and “passes_guidelines” tags, we will promote the module for inclusion in Ansible Extras.  At this point, you will be expected to maintain the module by fixing bugs and evaluating pull requests in a timely manner. 

More information about the review process can be seen here: 
https://github.com/ansible/ansible-modules-extras/blob/devel/REVIEWERS.md

Thanks again for submitting your Ansible module!

**Comment by drzraf** [comment]
Does it support check-mode? If yes, how?

File: web_infrastructure/apache2_site.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licens)
+
+DOCUMENTATION = '''
+---
+module: apache2_site
+version_added: 1.9
```
Comment (robinro) [inline web_infrastructure/apache2_site.py:19]: This would be 2.2 or 2.3.

**Comment by robinro** [comment]
The module looks good to me. Things to extend:
- implement check mode; not sure how to best do this since a2ensite has no dry-run, so one would have to look in the filesystem and reimplement some part of a2ensite within the module.
- write integration tests (see https://github.com/ansible/ansible/blob/devel/test/integration/roles/test_apache2_module/tasks/actualtest.yml)

Since `a2ensite`, `a2enmod` and `a2enconf` are the same (under debian/ubuntu), one could also consider to do a combined ansible module `apach2_settings` with `type = {site, mod, conf}`.
On the other hand: Suse has it's own implementation of `a2enmod` and doesn't have the other tools.

**Comment by drzraf** [comment]
@ansibot please read https://github.com/ansible/ansible/pull/7521

**Comment by andig** [comment]
This repo should be accepting merged again? Any chance to get the site module merged?

**Comment by gundalow** [comment]
New modules need to go into https://github.com/ansible/ansible

**Pull request closed without merging**